expand_source.sh

```diff
@@ -1,26 +1,38 @@
+#!/bin/bash
+
+declare -A already_expanded
+
 function expand_source {
-    if [[ $# -ne 1 ]]; then
-	print_error "Bashminify error in \"expand_source\", nr. of arguments"
-    fi
-    
-    local filename=$1
-    if [[ ! -f "${filename}" ]]; then
-	print_error "File \"${input_file}\" does not exist."
-	exit 1
-    fi
+	if [[ $# -ne 1 ]]; then
+		print_error "Bashminify error in \"expand_source\", nr. of arguments"
```

Comment:
We are checking if `expand_source` is used correctly but don't do the same for other functions such as `check_eof`. It probably makes sense to either always do the check or to never do it.

Reply:
Considering this function is not meant to be exposed to the user, I decided it made more sense to not check the input and removed the verifications.

Reply:
oops, I only just realized now that I originally added this check :D upon the first review it looked like you added this check but it turns out you just changed the whitespace here, sorry for commenting this initially and thanks for the fix!